### ISSUE TITLE
OpenShift v3.10 Logging (tech preview) cannot be installed 

### ISSUE THREAD

**Issue by FakeEmperor** [openshift/openshift-ansible#9702]
#### Description

OpenShift v3.10 Aggregated Logging cannot be installed due to several reasons:

- After image names refactoring (commit: https://github.com/openshift/openshift-ansible/commit/a310ad860090d050b9c71a54ea0b1cc38f104db4) we cannot set image version overrides
- OpenShift Aggregated logging images for ES5.x have released only one week ago and contain only the `latest` tag but `openshift-ansible` renders all its `DeploymentConfig`s using common, `3.10`, version.
- ConfigMap for ES5.x left the same as for 2.x and due to that ElasticSearch cannot start (for instance, there is no `kubernetes` cloud provider in ES5.x image for ElasticSearch)

`DeploymentConfig` for Kibana and ElasticSearch as well as Fluentd cannot deploy.

##### Version

Ansible version:

```
ansible 2.6.0
  config file = None
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Apr 11 2018, 07:36:10) [GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]
```

Git version:

`e055da131bf7cf899768c21fcec27f92bc16cb1d`

##### Steps To Reproduce
1. Set `openshift_logging_install_logging: True`
2. Set `openshift_logging_es5_techpreview: True`
3. Run either the `./openshift-ansible/playbooks/deploy-cluster.yml` or `./openshift-ansible/playbooks/openshift-logging/config.yml` 

##### Expected Results

OpenShift Aggregated Logging correctly rolls out.

##### Observed Results
OpenShift Aggregated logging is unable to rollout.

```
  1. Hosts:    kube-master01
     Play:     OpenShift Aggregated Logging
     Task:     Evaluating status of rolled out pod
     Message:  Node logging-es-data-master-amcfk74q in cluster logging-es was unable to rollout. Please see documentation regarding recovering during a rolling cluster restart.
```

##### Additional Information

If we build all the images ourselves (and `openshift_logging_pull_secret` with `openshift_logging_image`) or `openshift_logging_elasticsearch_image`, `openshift_logging_kibana_image` then it rolls out

* OS version: RHEL CentOS 7.4

**Comment by vrutkovs** [comment]
>OpenShift Aggregated logging images for ES5.x have released only one week ago and contain only the latest tag but openshift-ansible renders all its DeploymentConfigs using common, 3.10, version.

So, it just the missing images? This is not an openshift-ansible issue.

>ConfigMap for ES5.x left the same as for 2.x and due to that ElasticSearch cannot start (for instance, there is no kubernetes cloud provider in ES5.x image for ElasticSearch)

What's missing there? cc @ewolinetz

**Comment by ewolinetz** [comment]
This was fixed for 3.11 [1], this logic should have been removed from 3.10 as we did not support an es5 installation even as tech preview.

[1] https://github.com/openshift/openshift-ansible/commit/99ba1dd202ee429185541e2379526dab8c798c36
